api/lib/notifications/notificationController.js

```diff
@@ -26,17 +30,28 @@ class NotificationController {
 
   listenToNotifications() {
     eventSubscription((notification) => {
-      this.onNotification(notification)
+      this.onNotificationSendEMails(notification)
```

Comment:
it actually sends only one, right?

Reply:
well one per person

Reply:
I think it should be called `this.onNotificationSendEmail` since inside the function you are calling is in fact `sendEmail`. It is only one sent with each call

Reply:
not true XD it's sending one email per person

Reply:
`onNotificationSendEmails`

Reply:
feel free to edit that